contrib/julia-mode.el

```diff
@@ -131,6 +131,23 @@ This function provides equivalent functionality, but makes no efforts to optimis
              "\\\\"))
         (group "'"))))
 
+(defconst julia-hanging-assignment-regexp
+  (concat "^[^#\n]+ "
+          (regexp-opt '("=" "+=" "-=" "*=" "/=" "//=" "\\=" "^=" "÷=" "%=" "|=" "&=" "$=" "<<=" ">>="
+                        ".+=" ".-=" ".*=" "./=" ".//=" ".\\=" ".^=" ".÷=" ".%="))
+          (regexp-opt '(" #" " \n" "#" "\n"))))
+
+(defconst julia-hanging-operator-regexp
+  (concat "^[^#\n]+ "
+          (regexp-opt '("^" ".^" "//" ".//" "*" "/" "%" "&" "\\" ".*" "./" ".%" ".\\"
+                        "<<" ">>" ">>>" ".<<" ".>>" ".>>>"
+                        "+" "-" "|" "$" ".+" ".-"
+                        ":" ".." "|>"
+                        ">" "<" ">=" "<=" "==" "===" "!=" "!==" "<:" "≠" "≤" "≥"
+                        ".>" ".<" ".>=" ".<=" ".==" ".!="
+                        "&&" "||" "?"))
+          (regexp-opt '(" #" " \n" "#" "\n"))))
```

Comment:
Sorry, I should have explained. The current behavior is to increase the indent level if the previous line ends in a "=". In this PR I kept that behavior but extended it to all assignment operators. For example,
```julia
a =
    b = 
        1
```

For other operators, I increase the indent after the first one, but keep the indentation the same after that. For example,
```julia
a = 1 +
    2 +
    3 +
    4
```

My question is is the current behavior important? Is there something wrong with this for example?

```julia
a =
    b = 
    c
```
It seems strange to worry too much about chained assignments, but then again I don't have much experience with writing julia yet.